ArduCopter/toy_mode.cpp

```diff
@@ -990,6 +990,17 @@ void ToyMode::thrust_limiting(float *thrust, uint8_t num_motors)
     }
     uint16_t pwm[4];
     hal.rcout->read(pwm, 4);
+
+// @LoggerMessage: THST
+// @Description: Maximum thrust limitation based on battery voltage
```

Comment:
I wonder if we should tack on toymode here somewhere.  Otherwise we might end up with people asking about this one when they're rather unlikely to be able to get it.

Reply:
"Maximum thrust limitation based on battery voltage in Toy Mode"? Does that sound better?

Reply:
Sounds good.